### DISCUSSION TITLE
[HW4] regarding question 3.1

### DISCUSSION THREAD

**Discussion by sorobedio** [seominjoon/kaist-ai605#163] [Q&A] [answered]
In problem 2 the model used  in the documentation is BertForSequenceClassification. 
Since we are just applying that model to our data then i consider that in problem 2 the model is "BertForSequenceClassification". 
In the reference link the model used is "BertForQuestionAnswering". 
So i am quite confused when professor said "used your bert-cased model" in problem 3.1.

Does it mean that we have to  use the BertForSequenceClassification. model for question answering?
can we use BertForQuestionAnswering--bert_cased  model?
or which model shall we use if none of those is correct?

**Comment by seominjoon** [comment, marked as answer]
Yes, you are allowed to use BertForQuestionAnswering because it is part of Hugging Face library.
